pom.xml

```diff
@@ -38,6 +38,7 @@
                 
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
+        <version>2.7.5</version>

```

Comment:
We should have all springframework.* in pom.xml of eiffel-parent and use ${springboot.versin} property to define version number. Now we have springframework.* dependencies in 3 pom.xml files. Any subsequent uplift will require again changes in all the POM files. The goal is to change springboot.version only in POM of eiffel-parent.

Reply:
Agree, this is kind ahh rough PR, if these changes are looks fine, i will uplift spring-boot version in remrem-parent.. we can inherit the version from there..